src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -460,9 +461,11 @@ module Base = struct
       ; source_bad_timing: 'bool (* Payment only *)
       ; receiver_exists: 'bool (* Create_account only *)
       ; not_token_owner: 'bool (* Create_account, Mint_tokens *)
-      ; token_auth: 'bool (* Create_account *) }
+      ; token_auth: 'bool (* Create_account *)
+      ; update_not_permitted: 'bool (* Payment, delegate *) }
```

Comment:
Should this include `Mint_tokens` too? Newly minted tokens can be placed in any account, so it's another vector to 'receive' tokens.